napari_plugin_manager/qt_plugin_dialog.py

```diff
@@ -876,13 +890,14 @@ def setup_ui(self):
         self.packages_filter.setPlaceholderText(trans._("filter..."))
         self.packages_filter.setMaximumWidth(350)
         self.packages_filter.setClearButtonEnabled(True)
+        # Throttle keystrokes so only one filter is done every 200ms
```

Comment:
```suggestion
```
This comment isn't correct (120 ms now) because of change above, so I made a suggestion to comment there instead and then we can remove this one.